api/python/t4/data_transfer.py

```diff
@@ -346,6 +347,7 @@ def delete_object(path):
         s3_client.head_object(Bucket=bucket, Key=key)  # Make sure it exists
         s3_client.delete_object(Bucket=bucket, Key=key)  # Actually delete it
 
+NO_OP_COPY_ERROR_MESSAGE = "An error occurred (InvalidRequest) when calling the CopyObject operation: This copy request is illegal because it is trying to copy an object to itself without changing the object's metadata, storage class, website redirect location or encryption attributes."
```

Comment:
line length?

Reply:
https://github.com/google/styleguide/blob/gh-pages/pyguide.md#32-line-length
specific exception for module-level constants

Reply:
If you have an alternate convention you want me to follow, that'd be no problem. I just couldn't remember what the options are for splitting a long string

Reply:
I'm going to go with parens + implicit string joining as per the style guide

Reply:
No, that;s fine. I'mm used to seeing this as string concated, but if its in the style guide and doesn't break pylint then it's cool.

Reply:
That link says "Long string module level constants not containing whitespace that would be inconvenient to split across lines such as URLs or pathnames." There's a bunch of whitespace in that message